src/wp-includes/class-wp-scripts.php

```diff
@@ -551,6 +551,33 @@ public function get_inline_script_data( $handle, $position = 'after' ) {
 		return trim( implode( "\n", $data ), "\n" );
 	}
 
+	/**
+	 * Gets unaliased dependencies.
+	 *
+	 * An alias is a dependency whose src is false. It is used as a way to bundle multiple dependencies in a single
+	 * handle. This in effect flattens an alias dependency tree.
+	 *
+	 * @since 6.3.0
+	 *
+	 * @param string[] $deps Dependency handles.
+	 * @return string[] Unaliased handles.
+	 */
+	private function get_unaliased_deps( array $deps ) {
+		$flattened = array();
+		foreach ( $deps as $dep ) {
+			if ( ! array_key_exists( $dep, $this->registered ) ) {
+				continue;
+			}
+
+			if ( $this->registered[ $dep ]->src ) {
+				$flattened[] = $dep;
+			} elseif ( $this->registered[ $dep ]->deps ) {
+				array_push( $flattened, ...$this->get_unaliased_deps( $this->registered[ $dep ]->deps ) );
```

Comment:
Makes me so happy that the `...` spread operator is supported in PHP 5.6+.

Reply:
Ya, this is great. Worth noting - for reference sake - that implementation of the spread operator differs from 5.6 - 7.x in the sense that 5.6 _only_ supports argument unpacking. Thankfully, that is all that's needed here :)